### ISSUE TITLE
Android status bar app.json not working

### ISSUE THREAD

**Issue by Svarto** [expo/expo-cli#3832]
Hi, when trying to set the androidStatusBar variables in app.json it works as intended on Expo build but on EAS build doesn't work to make it transparent. When you set it with the defaults then it has the primary color, and when translucent it defaults to semi transparent. Is this a known issue? How to make it transparent as it is with Expo?

The default status bar (with no config in app.json) looks like this (was previously translucent and transparent as default in Android) in EAS:
![Screenshot_20210831-164249.jpg](https://user-images.githubusercontent.com/24279217/131572927-e4f4ddb6-3754-4ebc-9d5e-73ff63bc43c0.jpg)

**Comment by EvanBacon** [comment]
@Svarto we recently added some changes to the default template that's versioned with Expo SDK 42. Can you confirm that your project is using Expo SDK 42? If you could provide a set of steps or a minimum repro I'd be happy to investigate further!

**Comment by Svarto** [comment]
@EvanBacon thanks for the reply. Yes I'm on SDK42. I can try and see if I have time to create an example repo for you.

I think this might be connected with this issue I opened, seems like the template for production apps is different for custom dev clients when applying androidStatusBar specifications. https://github.com/expo/expo/issues/14308

**Comment by EvanBacon** [comment]
@Svarto, yes it looks like it may require `expo-system-ui` https://github.com/expo/expo/discussions/14286 in order to function properly.

**Comment by Svarto** [comment]
Issue still remains in SDK44

**Comment by Svarto** [comment]
Issue remains

**Comment by Svarto** [comment]
@EvanBacon following up here, any progress on getting the same outcome on both production and dev-client builds? This issue still remains...

**Comment by brentvatne** [comment]
@Svarto - if you use the `<StatusBar />` component at the root of your app it will have the same behavior in expo go and eas build. see https://github.com/brentvatne/svarto where i forked your repo, updated it to sdk 45, and added status bar component

**Comment by Svarto** [comment]
Hi @brentvatne , I found a solution and twin issue in the `expo/expo` repo, I managed to fix this through a config-plugin. 

The issue is that when the app launches, the status bar is not translucent as it was in expo build (and as it is on iOS). This is what causes the "jumping" in my example repo and this is not fixed by using `<StatusBar />` as this only does anything AFTER the splash screen has finished and the app has been loaded.

I will close this issue now, as there is an issue open in `expo/expo` on this. I will link from that issue to this one, to help with debugging. Thanks a lot for your help here!!

Twin issue: https://github.com/expo/expo/issues/11604#issuecomment-1018355394
